submission.py

```diff
@@ -32,6 +34,9 @@ def _planned_action(game, state, player):
         heuristic_func=heuristic,
         order_actions_func=order_actions,
     )
+    _LOGGER.debug(f"Selected action: {action}.")
```

Comment:
i believe i suggested before to log to selected action but i meant within the iterativ deepening as the result for every iteration of the minimax. the final action is logged anyways in the `act` function.

Reply:
im trying to say that this log should disappear and instead we should log it within the minimax (if that is not already the case). :D

Reply:
also this implies that you can just return at this point.

Reply:
Right! I misunderstood